usaspending_api/api_contracts/contracts/v2/agency/toptier_code/program_activity.md

```diff
@@ -0,0 +1,94 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List Program Activity [/api/v2/agency/{toptier_code}/program_activity/{?fiscal_year,filter,order,sort,page,limit}]
+
+Returns a list of Program Activity in the Agency's appropriations for a single fiscal year
```

Comment:
I think this would read better as `Program Activities` plural

Reply:
Good point, will make the change.